kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java

```diff
@@ -47,7 +49,10 @@
                 @TemplateParameter(name = "KIE_SERVER_PASSWORD", value = "${kie.password:Redhat@123}")
         }
 )
-public class DecisionServerBasicMulltiContainerTest extends DecisionServerBasicTest {
+@OpenShiftResources({
+        @OpenShiftResource("https://raw.githubusercontent.com/${template.repository:jboss-openshift}/application-templates/${template.branch:master}/secrets/decisionserver-app-secret.json")
```

Comment:
Why use double indent for annotation parameters, when using single indent everywhere else? (several occurrences, this is a sample).